app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/LiabilityDetails.scala

```diff
@@ -23,10 +23,15 @@ import uk.gov.hmrc.plasticpackagingtax.registration.views.model.TaskStatus
 case class LiabilityDetails(
   weight: Option[LiabilityWeight] = None,
   startDate: Option[Date] = None,
-  isLiable: Option[Boolean] = None
+  isLiable: Option[Boolean] = None,
+  willProcessMoreWeight: Option[Boolean] = None
```

Comment:
I wonder whether we could use a better name here? Does expectToExceedThresholdIn30d work better?